content/200-concepts/100-components/02-prisma-client/140-debugging.mdx

```diff
@@ -17,18 +17,15 @@ You can enable debugging output in Prisma Client via the [`DEBUG`](../../../refe
 Here are examples for setting these debugging options in bash:
 
 ```terminal
-# enable only `prisma:engine`-level debugging output
-export DEBUG="engine"
```

Comment:
Why remove this?

Reply:
It doesn't exist

Reply:
Could be changed to `export DEBUG="prisma:engine"` if we want to keep

Reply:
Restored